.codeclimate.yml

```diff
@@ -0,0 +1,14 @@
+---
+engines:
+  eslint:
+    enabled: false
+  fixme:
+    enabled: true
+  pep8:
+    enabled: true
+  radon:
+    enabled: true
+ratings:
+  paths:
+  - "**.py"
+exclude_paths: []
```

Comment:
I have it up, but it seems private so far. I'm also not sure it's the best tool to measure code quality, just gave it a try.